django/core/management/commands/diffsettings.py

```diff
@@ -8,26 +8,34 @@ def module_to_dict(module, omittable=lambda k: k.startswith('_')):
 
 class Command(BaseCommand):
     help = """Displays differences between the current settings.py and Django's
-    default settings. Settings that don't appear in the defaults are
-    followed by "###"."""
+    default settings."""
 
     requires_system_checks = False
 
     def add_arguments(self, parser):
         parser.add_argument(
             '--all', action='store_true', dest='all', default=False,
-            help='Display all settings, regardless of their value. Default values are prefixed by "###".',
+            help='Display all settings, regardless of their value. In "hash" mode, default values are prefixed by "###".',
         )
         parser.add_argument(
             '--default', dest='default', metavar='MODULE', default=None,
             help=(
                 "The settings module to compare the current settings against. Leave empty to "
                 "compare against Django's default settings."
+        )
```

Comment:
The flake8/test failure is because the above `help` is missing a closing parenthesis, hence a `SyntaxError`. You can always repush to the branch to rerun the tests.